### DISCUSSION TITLE
AWS re:invent - Well-Architected in ML

### DISCUSSION THREAD

**Discussion by rsiripan** [aws/amazon-sagemaker-examples#3059] [Q&A] [answered]
Hi,

I was on your session last week on well-architected in ML, can you share me with the presentation?
ray.siripan@tui.com

**Comment by aaronmarkham** [comment, marked as answer]
I asked around for you and the reinvent presentations can't be redistributed. However, you might find the videos of the sessions posted on YouTube.
